client/my-sites/site-settings/delete-site-options/index.jsx

```diff
@@ -102,45 +97,37 @@ module.exports = React.createClass( {
 					onClick={ this.checkForSubscriptions }
 					className="delete-site-options__link">
 					<div className="delete-site-options__content">
-						<h2 className="delete-site-options__section-title">{ strings.deleteSite }</h2>
+						<p className="delete-site-options__section-title is-destructive">
+							{ strings.deleteSite }
+						</p>
 						<p className="delete-site-options__section-desc">
-							{ this.translate(
+							{ translate(
 								'All your posts, images, and data will be deleted. ' +
-								'And this site’s address ({{siteAddress /}}) will be lost.',
-								{
-									components: {
-										siteAddress: <strong>{ selectedSite.wpcom_url || selectedSite.slug }</strong>
-									}
-								}
+								'And this site’s address will be lost.'
```

Comment:
If we're changing site's address to site address above, we should probably change it here too?